src/samples/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@a2a-js/samples",
+  "version": "0.3.5",
+  "description": "Samples for @a2a-js/sdk",
```

Comment:
Let's remove name, version and description as we're not going to publish examples to npm.